packages/@aws-cdk/aws-eks/README.md

```diff
@@ -270,7 +270,7 @@ The following code defines an Amazon EKS cluster with a default Fargate Profile
 
 ```ts
 const cluster = new eks.FargateCluster(this, 'MyCluster', {
-  version: eks.KubernetesVersion.V1_16,
+  version: eks.KubernetesVersion.V1_18,
```

Comment:
seems like there are a bunch of examples within the readme. (`KubernetesVersion.V1_16` is referenced 4x and `KubernetesVersion.V1_17` is referenced 7x)

is there a specific reason to only update 2 of them?

Reply:
Good point. Might as well update all of them.

Reply:
Really appreciated! Let me do it!